src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -485,13 +375,10 @@ frmelmnts.lbl.sharelink = Share using  the link -
 frmelmnts.lbl.showLess = Show less
 frmelmnts.lbl.showingResults = Showing results
 frmelmnts.lbl.showingResultsFor = Showing results for {searchString}
-frmelmnts.lbl.signUp = Register
```

Comment:
frmelmnts.lbl.signUp = Register, this is in use. 
check otp-component and signup-component